.eslintrc.js

```diff
@@ -46,6 +46,57 @@ module.exports = {
         ],
       },
     },
+    {
+      files: ['@xen-orchestra/lite/scripts/release.mjs'],
+      globals: {
+        $: true,
+        fs: true,
+        question: true,
+        chalk: true,
+        cd: true,
+        within: true,
+        path: true,
+        __dirname: true,
+      },
+      rules: {
+        'n/shebang': 'off',
+        'n/no-extraneous-import': 'off',
+        'no-console': 'off',
+        'n/no-process-exit': 'off',
+      },
+    },
```

Comment:
I'm not fond of `zx` modifying so much the env but it is what it is.

What do you think of reflecting this in the file extension, like `.zx.mjs` and adding the override on this pattern?

Reply:
Not fond either, but probably because I prefer explicitness ^^
 
I like your suggestion to apply a `.zx.mjs` extension.

@pdonias what do you think since you made this script? Is it OK for you if I rename the file and apply the ESLint config to `*.zx.mjs` pattern?

Reply:
After taking a look further at the script and ZX documentation, I think we could handle things differently:
1. I think the shebang `#!/usr/bin/env zx` can be removed since the script is not self-executable and probably never will be.
2. We could add `/* eslint-disable no-console, n/no-process-exit, n/no-extraneous-import  */` at top of the file
3. `$`, `cd`, `chalk`, `fs`, `path`, `question` and `within` can all be imported explicitly from `zx`package.
4. `__dirname` being not available in ESM, `path.join(__dirname, 'agpl-3.0.txt')` could be replaced with `fileURLToPath(new URL('agpl-3.0.txt', import.meta.url))`

I think it would better than having a dedicated configuration in root ESLint config.

@julien-f @pdonias What do you think?

Reply:
Sounds good. I also tend to prefer explicitness so if the 2nd solution works, then it's perfect. And I confirm we'll probably never need the shebang for our use case.

Reply:
It appears `zx` script can be ran directly with `node` when 3 & 4 are done, in this case, I think we should change the shebang to `#!/usr/bin/env node`.

For 2, maybe we should update the CLI override to include files in `scripts` directories.

Reply:
I've made the changes I suggested.